sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -212,7 +209,11 @@ dif_rstmgr_result_t dif_rstmgr_software_reset_is_held(
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
-  *asserted = rstmgr_software_reset_is_held(base_addr, peripheral);
+
+  uint32_t bitfield =
+      mmio_region_read32(base_addr, RSTMGR_SW_RST_CTRL_N_REG_OFFSET);
+  bitfield_bit32_index_t index = rstmgr_regen_rstctrl_bit_index(peripheral);
+  *asserted = bitfield_bit32_read(bitfield, index);
```

Comment:
Shouldn't this be negated? (the bits in the register seem to be active low).

Reply:
Good catch, thank you! Done.